src/Tickets/Seating/app/frontend/session/style.pcss

```diff
@@ -1,4 +1,4 @@
-.event-tickets .tec-tickets-seating__timer {
+.tribe-common .tec-tickets-seating__timer {
```

Comment:
If there is only one place we use this, do we need the "parent" class?

Reply:
Nope, we removed the parent class as the timer is hooked into Woo cart and checkout pages. Those pages won't have our `.event-tickets` or any other generic parent classes.

Reply:
Nope, we removed the parent class as the timer is hooked into Woo cart and checkout pages. Those pages won't have our `.event-tickets` or any other generic parent classes.

Reply:
But you didn't remove it - you changed it. Swapped `.tribe-common` for `.event-tickets`
My question was: do we need `.tribe-common` or is this specific enough already we can just do `.tec-tickets-seating__timer`

Reply:
@Camwyn 
Yes, the styling for the timer is tied to having a parent named `.tribe-common` so I just added it later. Normally, `.event-tickets` are already wrapped inside `.tribe-common`.